Configuration/PyReleaseValidation/python/upgradeWorkflowComponents.py

```diff
@@ -58,27 +55,9 @@
 
 upgradeProperties[2017] = {
     '2017' : {
-        'Geom' : 'DB:Extended',
-        'GT' : 'auto:phase1_2017_realistic',
-        'Era' : 'Run2_2017',
-        'ScenToRun' : ['GenSimFull','DigiFull','RecoFull','HARVESTFull'],
-    },
-    '2017NewFPix' : {
-        'Geom' : 'Extended2017NewFPix',
-        'GT' : 'auto:phase1_2017_realistic',
-        'Era' : 'Run2_2017_NewFPix',
-        'ScenToRun' : ['GenSimFull','DigiFull','RecoFull','HARVESTFull'],
-    },
-    '2017HCALdev' : {
-        'Geom' : 'Extended2017dev',
-        'GT' : 'auto:phase1_2017_hcaldev',
-        'Era' : 'Run2_2017_HCALdev',
-        'ScenToRun' : ['GenSimFull','DigiFull','RecoFull','HARVESTFull'],
-    },
-    '2017AllNew' : {
         'Geom' : 'Extended2017new',
         'GT' : 'auto:phase1_2017_hcaldev',
```

Comment:
I agree that eventually the auto GT should be changed back to "auto:phase1_2017_realistic". In principle, either within or after this PR, it could be enough to make a new "realistic" with the exact same content as is now in "auto:phase1_2017_hcaldev" and continue from there.

Maybe AlCa has some opinion? @mmusich @franzoni 

To my knowledge the geometry is being addressed in issue #15985 (I don't have any additional information).